src/Page/AboutPage/styles.css

```diff
@@ -0,0 +1,45 @@
+.about-page {
+  background-color: #fff;
+  text-align: center;
+}
+
+.page-section {
+  margin: 40px 0;
+}
+
+.about-page-content {
+  margin: 80px 0;
+  text-align: left;
+}
+
+.about-content {
+  margin: 30px 0;
+  text-align: left;
+}
+.mou-pdf {
+  cursor: pointer;
+  font-weight: 600;
+}
+.mou-pdf:hover {
+  text-decoration: underline;
+}
+@media screen and (max-width: 667px) {
+  .underliner {
+    margin-left: 150px;
+    width: 70px;
+  }
```

Comment:
There are a lot of files here but my search for `underliner` turned up nothing.  Is this rule needed?  If so, a left-margin of `150px` could be problematic for small screens.